lte/gateway/c/core/BUILD.bazel

```diff
@@ -524,15 +524,15 @@ MME_HDRS = [
     "oai/include/ip_forward_messages_def.h",
     "oai/include/ip_forward_messages_types.h",
     "oai/include/map.h",
-    "oai/include/messages_def.h",
-    "oai/include/messages_types.h",
-    "oai/include/mme_app_desc.h",
-    "oai/include/mme_app_messages_def.h",
-    "oai/include/mme_app_messages_types.h",
+    "oai/include/messages_def.hpp",
```

Comment:
messages_def.h and messages_types.h contain header files specific itti messaging. I think it's better to retain these as .h files only. Because itti library is built in 'C'

Reply:
I think these files can be re-used to define messages/types even though the lib changes in the future. Hence changed to hpp